### PR TITLE
Speed up build-containers job

### PR DESCRIPTION
**Pull request by dgn**
This will now only build one container to test the docker setup. Should be enough to prove that docker-in-docker still works.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -29,5 +29,5 @@ lint:
 
 # this will build the containers and then try to use them to build themselves again, making sure we didn't break docker support
 build-containers: maistra-builder
```
Comment (jwendell) [inline Makefile:31]: don't you have to change the target above too?

Comment (jwendell) [reply Makefile:31]: or... actually the dependency

Comment (dgn) [reply Makefile:31]: no. we still want to build all containers. what's changing is how many containers we build to test docker-in-docker

**Pull request merged**